pulumi/python/kubernetes/certmgr/__main__.py

```diff
@@ -40,7 +40,7 @@ def add_namespace(obj):
     chart_name = 'cert-manager'
 chart_version = config.get('chart_version')
 if not chart_version:
-    chart_version = 'v1.9.1'
+    chart_version = 'v1.10.0'
```

Comment:
Not necessary to adjust in this PR, but in my opinion we should check that the user has specified versions up front and crash with a helpful error message before anything is created. Defaults help with the "plug and play" ability of the example but I think we can have the defaults encoded in the `.example` as above and not hidden in a python script.

Thoughts on this? Do you think it would be tough to do?

Reply:
If we crashed here, would we wind up with any half-created state?

Reply:
Definitely, if we crash out things are really pear-shaped.

The long-term plan - and I think there may be an issue with it; if not I will add one - is to remove the fallbacks in the code for versions and force them all through the config. This requires that we make sure we tell the users how to add these values, and we probably should provide a "just give me all the known working values" option.